app/repositories/PublishedRepository.scala

```diff
@@ -48,32 +45,32 @@ class PublishedRepositoryImpl @Inject() (mongoComponent: ReactiveMongoComponent)
     )
     with PublishedRepository {
 
-  //TODO: Reactivate test coverage when both methods in use
-
-  //$COVERAGE-OFF$
   def save(id: String, process: JsObject): Future[RequestOutcome[String]] = {
 
     logger.info(s"Saving process $id to collection published")
 
-    val document: PublishedProcess = PublishedProcess(id, process)
+    val selector = Json.obj("_id" -> id)
+    val modifier = Json.obj("$inc" -> Json.obj("version" -> 1), "$set" -> Json.obj("process" -> process, "datePublished" -> LocalDateTime.now()))
```

Comment:
As with ApprovalRepository is this what we want to do?

Reply:
Yes, I think we want to know how many versions have actually been published - as the version on the approval process does not relate to the number of published processes.  So each time we publish a new version of the process, the version of the published process should be updated